hack/ocm-minikube-ramen.sh

```diff
@@ -35,11 +35,11 @@ minio_undeploy()
 	kubectl --context $1 delete -f $ramen_hack_directory_path_name/minio-deployment.yaml
 }
 exit_stack_push unset -f minio_undeploy
-ramen_image_directory_name=localhost
-ramen_image_name=ramen-operator
-ramen_image_tag=v0.N
+ramen_image_directory_name=${ramen_image_directory_name-localhost}
+ramen_image_name=${ramen_image_name-ramen-operator}
+ramen_image_tag=${ramen_image_tag-v0.N}
 ramen_image_name_colon_tag=${ramen_image_directory_name}/${ramen_image_name}:${ramen_image_tag}
-exit_stack_push unset -v ramen_image_name_colon_tag ramen_image_tag ramen_image_name ramen_image_directory_name
+exit_stack_push unset -v ramen_image_name_colon_tag
```

Comment:
(nit) Assuming `ocm-minikube-ramen.sh` is run in a separate shell (e.g. without `. ` prefix), I think it is ok to unset these variables.